src/darwin/Framework/CHIP/MTRDeviceControllerDelegate.h

```diff
@@ -28,6 +28,25 @@ typedef NS_ENUM(NSInteger, MTRCommissioningStatus) {
     = 3,
 } API_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4));
 
+/**
+ * Read from controllee's Information  at Commissioning
+ */
+MTR_NEWLY_AVAILABLE
+@interface MTRReadCommissioningInfo : NSObject
+
+/**
+ * Controllee's VendorID Attribute of Basic Information cluster
+ */
+@property (nonatomic, copy, readonly, nonnull) NSNumber * vendorID MTR_NEWLY_AVAILABLE;
```

Comment:
```suggestion
@property (nonatomic, copy, readonly) NSNumber * vendorID MTR_NEWLY_AVAILABLE;
```
nonnull is the default.